demo/react/components/notification/default.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { Fragment, useContext } from 'react';
+
+import { Button, NotificationTheme } from 'LumX';
+import {
+    NotificationProvider,
+    NotificationState,
+    notificationContext,
+} from 'LumX/components/notification/react/NotificationProvider';
+
+/////////////////////////////
+
+interface IProps {
+    /**
+     * The theme to use to display this demo.
+     */
+    theme: NotificationTheme;
+}
+
+/////////////////////////////
+
+/**
+ * The demo for the default <Notification>s.
+ *
+ * @return {React.ReactElement} The demo component.
+ */
+const DemoComponent: React.FC<IProps> = ({ theme }: IProps): React.ReactElement => (
+    <Fragment>
+        <NotificationProvider>
```

Comment:
Not sure I understand how it should implemented in a real application. I this provider should be included at the root of the UI tree ?

Reply:
Yep, like other providers